sources/common-dylan/tests/regressions.dylan

```diff
@@ -29,7 +29,19 @@ define test bug-5954 ()
               -17);
 end test bug-5954;
 
+define test bug-github-1239 ()
```

Comment:
Bike shedding: to me it would be better to name the test for what it does, e.g. test-shift-left-signals-overflow, and put the bug URL in a comment. Up to you though.

Reply:
Fair enough. Updated again.